### PR TITLE
Ui refactoring backend

### PR DESCRIPTION
**Pull request by taaraora**
1. GetPluginConfig handler was added
2. Analyze core at bootstrap process in case of plugins config absence saves default one.

### CODE REVIEW

File: ui/proxy.conf.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "/api": {
-    "target": "http://ec2-34-249-11-236.eu-west-1.compute.amazonaws.com:31834",
+    "target": "http://54.183.122.86:32291",
```
Comment (stgleb) [inline ui/proxy.conf.json]: what is a purpose of this IP address?

Comment (taaraora) [reply ui/proxy.conf.json]: that will be removed at PR - https://github.com/supergiant/analyze/pull/42

Comment (taaraora) [reply ui/proxy.conf.json]: > Generally looks good, but I would wait for @josephkane to approve

@josephkane is working on exactly the same files, and he changes them a lot, we don't need to review UI part here. UI part is reviewed at  https://github.com/supergiant/analyze/pull/42

Comment (stgleb) [reply ui/proxy.conf.json]: Ok, thank you

**Pull request merged**